model/schema/allele.yaml

```diff
@@ -207,6 +356,13 @@ classes:
       object:
         range: CellLine
 
+  AlleleCellLineAssociationDTO:
+      is_a: AuditedObjectDTO
+      slots:
+        - allele_curie
+        - predicate_name
+        - cell_line_curie
+
```

Comment:
because this is not inheriting from an 'evidence' slot-bearing class (like a hypothetical AssociationDTO class), 'evidence_curies' needs to be added here